test/unit/installTest.js

```diff
@@ -0,0 +1,49 @@
+'use strict'
+
+const request = require('supertest');
```

Comment:
code style seems different in this file, the other file you were changing isn't using semi colons

Reply:
that is true. I changed it to use semi colons to be consistent with other test files that were there :-)

Reply:
hah fine, whole repo should get a once over separately then